scripts/artifactVariables.sh

```diff
@@ -17,9 +17,12 @@ COMMIT_MESSAGE=${COMMIT_MESSAGE//\"/\\\"}
 function submitPreviewComment {
     COMMENT_JSON="{\"body\": \"$1\"}"
 
-    if PR_NUMBER=$(basename $CI_PULL_REQUEST); then
+    if [ -n "$PR_NUMBER" ] ; then
+        UPSTREAM_USERNAME=$(echo $CI_PULL_REQUEST | awk -F/ '{print $4;}')
+        UPSTREAM_REPONAME=$(echo $CI_PULL_REQUEST | awk -F/ '{print $5;}')
+        UPSTREAM_API_BASE_URL="https://$GH_AUTH_TOKEN:$GH_API_URL/repos/$UPSTREAM_USERNAME/$UPSTREAM_REPONAME"
```

Comment:
seems like all of this can be computed at the top of the file with the other variables.

then it can also be used in the `else` block on line 29 for consistency.